lib/cyclone-core/src/component_view.rs

```diff
@@ -56,14 +58,12 @@ impl Default for ComponentViewWithGeometry {
 pub struct ThisComponent {
```

Comment:
This can go away, and the caller can just use `ComponentViewWithGeometry` now :)